script/dumpdb

```diff
@@ -1,5 +1,33 @@
 #!/bin/bash
 set -e
-# https://www.coderedcorp.com/blog/how-to-dump-your-django-database-and-load-it-into-/
-docker-compose run --rm web python manage.py dumpdata --natural-foreign --natural-primary -e contenttypes -e auth.Permission --indent 2 -o /tmp/dumpdb.json
-echo "Database dump written to /tmp/dumpdb.json"
+
+if [ "$1" = "testfixtures" ]; then
+  # Dump out updated test testfixtures
+  echo -e "\033[0;31mYou are about to replace the existing test fixtures with the current content of your local database.\033[0m"
+  read -p "Are you sure you want to do this? (y/n):" -n 1 -r
+  echo    # (optional) move to a new line
+  if [[ ! $REPLY =~ ^[Yy]$ ]]
+  then
+      exit 1
+  fi
+  docker-compose run --rm web python manage.py purge_revisions
+  echo "Purged old page revisions from database"
+  docker-compose run --rm web python manage.py dumpdata \
+    --natural-foreign --natural-primary \
+    -e contenttypes \
+    -e auth.permission \
+    -e wagtailcore.pagelogentry \
+    --indent 2 \
+    -o fixtures/testdata.json
+  echo "Exported updated test fixtures"
+else
+  # Dump out the database to /tmp/
+  # https://www.coderedcorp.com/blog/how-to-dump-your-django-database-and-load-it-into-/
+  docker-compose run --rm web python manage.py dumpdata \
+    --natural-foreign --natural-primary \
+    -e contenttypes \
+    -e auth.permission \
+    --indent 2 \
+    -o /tmp/dumpdb.json
+  echo "Database dump written to /tmp/dumpdb.json"
+fi
```

Comment:
could reduce duplication by saving the filename to an environment variable and running regardless of textfixtures setting.